coprocess/proto/Taskfile.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+
+tasks:
+  install-deps:
+    desc: install the required dependencies to generate the bindings. gRPC is required to be installed
+    cmds:
+      - go get -u github.com/golang/protobuf/protoc-gen-go
+      - pip3 install grpcio grpcio-tools
+  generate:
+    desc: Generating bindings for Go, Python and Ruby
+    cmds:
+      - protoc --go_out=. --go-grpc_out=. --go-grpc_opt=require_unimplemented_servers=false *.proto
```

Comment:
I think it would be a good idea to add a log here:
"Generating bindings for Go."
just to keep it coordinated with the other ones. What do you think?

Reply:
done! ✅